docs/docs/images/kpops-components_class-diagram.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-03-13T10:59:02.923Z" agent="5.0 (X11)" etag="O9JJTFi92rp0hwrreF6r" version="21.0.6" type="device"><diagram id="C5RBs43oDa-KdzZeNtuy" name="Page-1">7ZrJdpswFIafxsv0MJsu4ylje9I6qZulDNdGtUBUCA99+gojJuOmdpJSJ/Yqur8kJO7wCYhbetdfXjAUep+oC6SlKe6ypfdamqYauiH+JMpKKrbVTpUpw67UCmGIf4EUFanG2IWoMpBTSjgOq6JDgwAcXtEQY3RRHTahpLpqiKZQE4YOInV1hF3upaqttQv9EvDUy1ZWrY9pj4+ywfJOIg+5dFGS9H5L7zJKedryl10gifcyv4yuViNyO7Murr9EP9FD5+b+87ez9GKDfabkt8Ag4M+/dHT7dRbiiwd3dX9N7tvXoyvrTLXkvfFV5jBwhf+kSRn36JQGiPQLtSMcESa9MmSUCY3ROHAhWUoRVjHvltJQiKoQfwDnK5kgKOZUSB73ieyFJebfk+kfTGk9yosl7d6ybKykQdAYSAc5s+l6+S4lyV56LkxQTISnOhMa8AHyMUlmXAKZA8cOkh1yK6oq7fp0CNzzJAWFOCbUmaXSABMiNzARzWxaS9PHqutOEj3ijM6g1GMjZDnjvCfLRC3bivT3rpGW4yIaMweeCK/MXI7YFPgT47R0XBL6UtXIPLoA6gNnKzGAAUEcz6ulhWSFTvNxRRaKhkzEPZJS7nqOSCxXusMhEBwIj/ohDRLfbGZtNf8WHuYwDNHaOQtBtmquPT88tZD3zvv2oCv0/SI3B8Zh+aSvZa+e4ScDcGYvCprlmlcimaH8o/BkZN+LGf+FD3+t30OpU/W1609OvaM4qZU8l+yNXLI2ciQFhZy1kSb5Nl6QObXCbmnWmrQunovmNGnexGNgAXCIzsMw6xfLlYYcT/UbRjVieVWXq19psvq1Wgw7KIJeemZGrwToP7m5oXrdg86KWY+H2SiN1ZqrTzR+GY31HZ+aXp3aL0oEvVaYN2gyQ2uIHg0uTbOCS92q47LdaHUaz6jO0/vVoRHB2JEI+kERwagRYcgZID86LiYo1UcoY8sjlN0kEz6+nQNb3G5pUmI+ZtdLjGLa2npzB725Y1krB1XW5vaDvpufFkdT2lp2vB7M25Gqnc77d3DeWzuCwTwoMFjbwTDEwewI4aDqVThsexdo+MOpfoLDO4BD+03CoV2Dw7Zvr2terB1QIsZxf4LV2jtwxGiUI+aJI++AI/aOHDmsjwp2jSN3jLqxA+yoviqYzX1VEGbxI5P0X3HFb3X0/m8=</diagram></mxfile>
```

Comment:
minor: add new line :D

Reply:
haha I think I shouldn't touch that file. It is generated by [drawio](https://app.diagrams.net/) and its only purpose is to load an existing diagram to edit it. A bit like `poetry.lock`